docs/jwst/mrs_imatch/LICENSE.rst

```diff
@@ -0,0 +1,5 @@
+*******
+LICENSE
+*******
+
+.. include:: ../../../LICENSE
```

Comment:
Not that I'm asking for a change here. We should in the future go over all documentation and files and make them "standard".